doc/release/RELEASE-NOTES.md

```
@@ -40,6 +55,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ### Release instructions:
 * Ensure that the following commands are included in the `post_deployment` command:
+    * `migrate`
```

Markdownlint (0.35.0)

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)